pkg/assert/meta.go

```diff
@@ -0,0 +1,65 @@
+package assert
+
+import (
+	"testing"
+
+	"github.com/giantswarm/sonobuoy-plugin/v5/pkg/capiutil"
+)
+
+func LabelIsSet(t *testing.T, object capiutil.TestedObject, label string) {
```

Comment:
Not strong request, but just a beauty thing: Could it make sense to re-define `TestedObject` interface in this package so that there wouldn't be coupling with `capiutil` when the interface itself is combined from upstream interfaces?